src/config/apollo.rs

```diff
@@ -15,27 +15,27 @@ pub struct Apollo {
     ///
     /// Setting `userVersion` for Apollo.
     #[serde(default = "default_user_version")]
-    pub user_version: String,
+    pub user_version: Option<String>,
```

Comment:
Serde default is already set. We can't have both serde default and `Option`, which defaults to `None`.